packages/error-reporting/bin/test.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
```

Comment:
These tests exclude Windows environments. Just running `npm test`:

```
> @google-cloud/error-reporting@0.1.0 test C:\Users\sawch\dev\google-cloud-node\packages\error-reporting
> $(npm bin)/nyc --exclude="fuzzer.js" $(npm bin)/mocha ./test/unit/*.js

'$' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! Test failed.  See above for more details.
```

We use https://github.com/shelljs/shelljs to get around this.